app/src/main/java/org/wikipedia/compose/extensions/Modifier.kt

```diff
@@ -60,6 +63,16 @@ fun Modifier.pulse(
     }
 }
 
+fun Modifier.clickableWithRipple(
+    onClick: () -> Unit
+): Modifier = composed {
```

Comment:
Are we totally sure we need this?
As I understand it, we would just need to add `indication = ripple()` wherever we want a ripple.
This seems to instantiate quite a bit of additional logic (another `composed{}` block, and another `MutableInteractionSource`), which feels unnecessary.